examples/client/findFirstFinalizedBlockNoLaterThan.ts

```diff
@@ -0,0 +1,67 @@
+import { createConcordiumClient } from '@concordium/node-sdk';
+import { credentials } from '@grpc/grpc-js';
+
+import meow from 'meow';
+
+const cli = meow(
+    `
+  Usage
+    $ yarn run-example <path-to-this-file> [options]
+
+  Required
+    --time,     -t  The time to search for, specified as ms since UNIX epoch.
+
+  Options
+    --help,         Displays this message
+    --from,         Block height to start searching from. Defaults to 0.
+    --to,           Block height to end search at. Defaults to latest finalized.
+    --endpoint, -e  Specify endpoint of the form "address:port". Defaults to localhost:20000
+`,
+    {
+        importMeta: import.meta,
+        flags: {
+            time: {
+                type: 'number',
+                alias: 't',
+                isRequired: true,
+            },
+            from: {
+                type: 'number',
+            },
+            to: {
+                type: 'number',
+            },
+            endpoint: {
+                type: 'string',
+                alias: 'e',
+                default: 'localhost:20000',
+            },
+        },
+    }
+);
+
+const [address, port] = cli.flags.endpoint.split(':');
+const client = createConcordiumClient(
+    address,
+    Number(port),
+    credentials.createInsecure()
+);
+
+/**
+ * Find the first block finalized after the time specified by "--time"
+ */
+
+(async () => {
+    const from =
+        cli.flags.from !== undefined ? BigInt(cli.flags.from) : undefined;
```

Comment:
```suggestion
        cli.flags.from ? BigInt(cli.flags.from) : undefined;
```

Reply:
I don't agree with these changes, as this type loose of comparison is less precise and can easier lead to errors when changing things 🙂